responses.go

```diff
@@ -14,6 +14,9 @@ type Response interface {
 	// example.
 	SetTemplateGroup(name string)
 
+	// SetHeader sets the HTTP header that will be sent with the response.
```

Comment:
We could warn that a SetHeader method overrides existing headers with the same key.